app/views/profiles/edit.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>Profiles#edit</h1>
-<p>Find me in app/views/profiles/edit.html.erb</p>
+<h1>Hello <%= @profile.first_name %></h1>
+<%= simple_form_for(@profile) do |f| %>
+    <%= f.input :first_name %>
+    <%= f.input :last_name %>
+    <%= "Email: (not changeable) #{@profile.user.email}" %>
+    <%= f.input :address %>
```

Comment:
do you want to change the databse schema to be able to store the address with the city on a separate field ?

Reply:
let's keep the address as we have it for now